judges/quality-of-service/quality-of-service.rb

```diff
@@ -32,15 +32,26 @@
   total = 0
   success = 0
   duration = 0
+  mttr = {}
   Fbe.unmask_repos.each do |repo|
     Fbe.octo.repository_workflow_runs(repo, created: ">#{f.since.utc.iso8601[0..9]}")[:workflow_runs].each do |json|
+      workflow_id = json[:workflow_id]
+      if json[:conclusion] == 'failure'
```

Comment:
@Suban05 What if the status is `failure` several times in a row. E. g.:
| id | workflow_id | status  |
|----|-------------|---------|
| 1  | 1           | success |
| 2  | 1           | failure |
| 3  | 1           | failure |
| 4  | 1           | success |

`last_failed_at` - should be the first or last (id: 2 or 3 ) failure of workflow run?

Reply:
@Yegorov good point, thanks

Reply:
@Yegorov added changes and a test, take a look, please

Reply:
@Suban05 Probably here we need the opinion of @yegor256 , we must rewrite failed time or leave the first failed time?
I think we must leave the first failed time.

Reply:
I believe we must take the first failure. MTTR means "mean time to repair."